app/dts/bindings/behaviors/zmk,behavior-underglow-indicators.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Set underglow for num lock indicator
```

Comment:
inaccurate description. This file is more general and also used for the scroll-lock indicator and capslock indicator